test/unit/controllers/VatRegisteredUkControllerSpec.scala

```diff
@@ -54,6 +55,7 @@ class VatRegisteredUkControllerSpec extends ControllerSpec with BeforeAndAfterEa
   private val mockSubscriptionBusinessService = mock[SubscriptionBusinessService]
   private val mockSubscriptionDetailsService  = mock[SubscriptionDetailsService]
   private val mockSubscriptionFlow            = mock[SubscriptionFlow]
+  private val mockSubscrptionFlowLeft         = mock[SessionError]
```

Comment:
That's a misleading name that only makes sense with semantic context. If it's a mock of a SessionError, make it clear that's what it is in the name.

Though I don't think you *need* to mock it. Why not simply use an instance of the intended session error? Then you can test more precisely.